SoapyFCDPP/SoapyFCDPP.cpp

```diff
@@ -500,9 +574,33 @@ std::vector<double> SoapyFCDPP::listBandwidths(const int direction, const size_t
 }
 
 // Registry
+std::string findAlsaDevice(const char *hidpath)
+{
+    // Stolen from fcdctl, we locate the audio device for this HID interface by USB bus ID:
+    // https://github.com/phlash/fcdctl/blob/8f9e855db26f531c1e2af0dcd3648e566e7f05b9/main.c#L76
+    int usb1, usb2, n = sscanf(hidpath,"%x:%x", &usb1, &usb2);
+    assert(n==2);
+    for (n=0; n<16; n++) {
+        char aspath[32];
+        sprintf(aspath,"/proc/asound/card%d/usbbus", n);
+        FILE *fp = fopen(aspath,"r");
+        if (fp) {
+            fgets(aspath,sizeof(aspath),fp);
+            fclose(fp);
+            int asu1, asu2;
+            sscanf(aspath,"%d/%d", &asu1, &asu2);
+            if (asu1==usb1 && asu2==usb2) {
+                sprintf(aspath,"hw:%d,0",n);
+                return aspath;
+            }
+        }
+    }
+    return "not found";
+}
+
 SoapySDR::KwargsList findFCDPP(const SoapySDR::Kwargs &args)
 {
-    SoapySDR_log(SOAPY_SDR_TRACE, "findFCDPP");
+    SoapySDR_log(SOAPY_SDR_INFO, "findFCDPP");
```

Comment:
This is going to be a bit too chatty when this module is installed and someone is using lets say RTL. I would revert to SOAPY_SDR_TRACE

Reply:
Yep - discovered this once I got it going, will review other log level changes too.